### ISSUE TITLE
Script Brushes: Error creating a new folder with image:save()

### ISSUE THREAD

**Issue by gratural** [wowdev/noggit3#97]
> [error]: failed to save image with error 79 (in function img_save)
This error only occurs with image:save().
The append_file() function create files and folders perfectly.

For now, I've used this wonderful solution:
```
    append_file(png_full_name,"")
    png_image:save(png_full_name)
```

**Comment by ihm-tswow** [comment]
confirmed, the function doesn't even call mkdirs.

**Comment by ihm-tswow** [comment]
fix: 74309b7cd33e19ae2b9e69160612d98eae3b8dcc
